apis/externalsecrets/v1/secretstore_yandexlockbox_types.go

```diff
@@ -14,30 +14,16 @@ limitations under the License.
 
 package v1
 
-import (
-	esmeta "github.com/external-secrets/external-secrets/apis/meta/v1"
-)
-
-type YandexLockboxAuth struct {
-	// The authorized key used for authentication
-	// +optional
-	AuthorizedKey esmeta.SecretKeySelector `json:"authorizedKeySecretRef,omitempty"`
-}
-
-type YandexLockboxCAProvider struct {
-	Certificate esmeta.SecretKeySelector `json:"certSecretRef,omitempty"`
-}
-
 // YandexLockboxProvider Configures a store to sync secrets using the Yandex Lockbox provider.
 type YandexLockboxProvider struct {
 	// Yandex.Cloud API endpoint (e.g. 'api.cloud.yandex.net:443')
 	// +optional
 	APIEndpoint string `json:"apiEndpoint,omitempty"`
 
-	// Auth defines the information necessary to authenticate against Yandex Lockbox
-	Auth YandexLockboxAuth `json:"auth"`
+	// Auth defines the information necessary to authenticate against Yandex Certificate Manager
```

Comment:
It seems `Lockbox` (instead of `Certificate Manager`) should be here

Reply:
I also like `// Auth defines the information necessary to authenticate against Yandex Cloud` both in `secretstore_yandexcertificatemanager_types.go` and `secretstore_yandexlockbox_types.go`